packages/create-react-app/index.js

```diff
@@ -154,8 +154,8 @@ function getInstallPackage(version) {
 function getPackageName(installPackage) {
   if (~installPackage.indexOf('.tgz')) {
```

Comment:
Can you please kill `~` on this line as well? It is the kind of "clever code" that is extremely confusing 😄 . I missed it when I review the change adding it.